pkg/clusters/utils.go

```diff
@@ -75,3 +75,30 @@ func GetIngressLoadbalancerStatus(ctx context.Context, c Cluster, namespace stri
 		return nil, fmt.Errorf("%T is not a supported ingress type", ingress)
 	}
 }
+
+// CreateNamespace create customized namespace
+func CreateNamespace(ctx context.Context, cluster Cluster, namespace string) error {
+	nsName := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+
+	_, err := cluster.Client().CoreV1().Namespaces().Create(context.Background(), nsName, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("failed creating namespace %s, err %v", namespace, err)
+	}
```

Comment:
Do you want to tolerate if the namespace already exists, or no? We should probably be more explicit about this, and add comments + cleanup code.

Currently this does NOT tolerate the existence of the namespace, which honestly I think is the way to go for now: it precludes the notion of having to deal with colliding resources or existing resources from a previous / alternative deployment and creates a cleanroom environment for the addon every time.

Reply:
Assuming this is fresh installed KTF cluster, and the target namespace does not exist. Yeah, cleaning namespace is a valid suggestion.